update readme for AdditionalTranslationsComponentSlot (#2321)
openedx/frontend-app-authoring: src/plugin-slots/AdditionalCourseContentPluginSlot/README.md

### Slot ID Aliases

> * `additional_course_content_plugin`

Reviewer: Out of scope for this PR, but this is a bad name. Shouldn't it be something like `pages_resources_content_permissions_additional_cards` ?

Reviewer 2: The *alias* can't change for backwards compatibility reasons, but I'm very open to changing the id and making the current id (`org.openedx.frontend.authoring.additional_course_content_plugin.v1`) another alias.

  
## Description

> This slot is used to add a custom block in the **Content permissions** section of the **page & resources** page.

Reviewer: Does anyone know why we even have "Content permissions" as a separate section? It doesn't seem that useful to me to have one or two cards separate from the others. Why are the permissions settings not just managed within the "regular" settings for the particular app?

> **AdditionalCourseContentPluginSlot** is dependent of [AdditionalCoursePluginSlot](../AdditionalCoursePluginSlot/README.md), to make it visible is also needed to provide settings for **AdditionalCoursePluginSlot**.

Reviewer: ```suggestion
**AdditionalCourseContentPluginSlot** is depends on [AdditionalCoursePluginSlot](../AdditionalCoursePluginSlot/README.md); to make it visible, you must also provide configuration for the **AdditionalCoursePluginSlot**.
```

This is weird to me. Why does one of these slots depend on the other? There isn't a technical reason as far as I can see, because if I do this:

```tsx
    'org.openedx.frontend.authoring.additional_course_content_plugin.v1': {
      plugins: [
        {
          op: PLUGIN_OPERATIONS.Insert,
          widget: {
            id: 'custom_additional_course_content_id',
            type: DIRECT_PLUGIN,
            RenderWidget: () => (
              <Card className={'shadow justify-content-between'} >
                    <Card.Header
                        title={'Additional Course Content'}
                    />
                    <Card.Body>
                        <Card.Section>
                          <p>Additional course contentfrom slot plugins. Add all the content needed here.</p>
                        </Card.Section>
                    </Card.Body>
                </Card>

            ),
          },
        },
      ],
    },
    'org.openedx.frontend.authoring.additional_course_plugin.v1': {
      plugins: []
    }
```
it works fine. Even though this is the same as not having an `additional_course_plugin` at all.

Reviewer: <img width="1287" height="1005" alt="Screenshot 2025-07-24 at 3 00 30 PM" src="https://github.com/user-attachments/assets/1342313c-a00c-4832-a9d8-cce55e10b346" />
